exercises/concept/annalyns-infiltration/.docs/hints.md

```diff
@@ -0,0 +1,11 @@
+# Hints
+
+## General
+
+- There are three boolean operators to work with boolean values: `not`, `&&`, and `||`.
+- Multiple operators can be combined in a single expression.
+
+## 1. Check if a fast attack can be made
+
+- The boolean operators can also be applied to boolean parameters.
```

Comment:
What do you mean by this? I am confused by both «parameters» and «also».

Reply:
Generally, operators also apply to arguments. In this case though the text is just left over from the Annalyn's exercise. Change or delete as needed to fit the narrative of Pacman Rules.